TrashMob/client-app/src/components/EventManagement/ShareToSocialsModal.tsx

```diff
@@ -0,0 +1,153 @@
+import { useState } from "react";
+import { Modal, Button, Tooltip, OverlayTrigger } from 'react-bootstrap';
+import { Clipboard, GeoAltFill, Clock } from "react-bootstrap-icons";
+import { FacebookShareButton, FacebookIcon, TwitterShareButton, TwitterIcon, LinkedinShareButton, LinkedinIcon, WhatsappShareButton, WhatsappIcon, EmailShareButton, EmailIcon } from 'react-share';
+import Card from 'react-bootstrap/Card';
+
+
+interface ModalProps {
+  createdEvent: any;
+}
+
+export const SocialsModal: React.FC<ModalProps> = (props) => {
+  const [show, setShow] = useState(true);
+  const [copiedLink, setCopied] = useState(false);
+  const eventLink = `${window.location.origin}/eventdetails/${props.createdEvent.id}`
+  const eventDate = new Date(props.createdEvent.eventDate).toLocaleDateString("en-us", { year: "numeric", month: "2-digit", day: "2-digit" })
+  const eventTime = new Date(props.createdEvent.eventDate).toLocaleTimeString("en-us", { hour12: true, hour: 'numeric', minute: '2-digit' })
+
+  const tooltip = (
+    <Tooltip id="tooltip">
+      Copied to clipboard!
+    </Tooltip>
+  );
+
+  const handleClose = () => {
+    setShow(false);
+  }
+
+  const handleCopyLink = () => {
+    navigator.clipboard.writeText(eventLink);
+    setCopied(true);
+    setTimeout(() => {
+      setCopied(false);
+    }, 2000)
+  }
+
+  const getShareMsgContent = (provider: string) => {
```

Comment:
The ShareUrl.tsx contains urls to share on twitter and facebook. Can you reuse that to keep the message consistent when share in different pages? 

Share button available on MyDashboard and event detail pages too. Possibly we can change the share button to this share dialog without event create information to keep consistence.  What do you think?